### ISSUE TITLE
NoSectionError when using :set proxy ?

### ISSUE THREAD

**Issue by The-Compiler** [The-Compiler/qutebrowser#1109]
```
Traceback (most recent call last):
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/config.py", line 649, in get
    sect = self.sections[sectname]
KeyError: 'proxy'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 227, in run_safely
    self.run(text, count)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 221, in run
    result.cmd.run(self._win_id, args)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/command.py", line 505, in run
    self.handler(*posargs, **kwargs)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/config.py", line 727, in set_command
    val = self.get(section_, option, transformed=False)
  File "/home/florian/proj/qutebrowser/git/.venv/lib/python3.4/functools.py", line 458, in wrapper
    result = user_function(*args, **kwds)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/config.py", line 653, in get
    raise configexc.NoSectionError(sectname)
qutebrowser.config.configexc.NoSectionError: Section 'proxy' does not exist!
```

**Comment by The-Compiler** [comment]
Similarly, there's a `NoOptionError` when doing the same with a known section, e.g. `:set general ?`

```
Traceback (most recent call last):
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/config.py", line 655, in get
    val = sect[optname]
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/sections.py", line 53, in __getitem__
    return self.values[key]
KeyError: ''

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 227, in run_safely
    self.run(text, count)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 221, in run
    result.cmd.run(self._win_id, args)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/command.py", line 505, in run
    self.handler(*posargs, **kwargs)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/config.py", line 727, in set_command
    val = self.get(section_, option, transformed=False)
  File "/home/florian/proj/qutebrowser/git/.venv/lib/python3.4/functools.py", line 458, in wrapper
    result = user_function(*args, **kwds)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/config.py", line 659, in get
    raise configexc.NoOptionError(optname, sectname)
qutebrowser.config.configexc.NoOptionError: No option '' in section 'general'
```

**Comment by The-Compiler** [comment]
This was fixed in 54e2cea460915102358733aee3ac1f5944dc153a.
